README.md

```diff
@@ -44,6 +44,8 @@ Learn More:
 
 Get the Latest News:
 
+- **facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
```

Comment:
Could you also please sync with https://github.com/mattermost/mattermost-server#get-the-latest-news?

Reply:
Sir updated https://github.com/mattermost/mattermost-server/pull/16186